deploy/apollo/apollo-rhworker/templates/cronjob.yaml

```diff
@@ -30,9 +30,11 @@ spec:
             imagePullPolicy: IfNotPresent
             command: ["/bin/bash", "-c"]
             args:
-            - |
```

Comment:
I might also suggest something like

```
spec:
  jobTemplate:
    spec:
      activeDeadlineSeconds: 300
```

which will autowhack jobs after 5 minutes - far in excess of what the normal runtime should be.

ref: https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup

Reply:
good idea. thank you!